server/cluster/cluster.go

```diff
@@ -108,3 +109,7 @@ func (c *Cluster) GetShards() []storage.ShardID {
 func (c *Cluster) GetShardNodes() metadata.GetShardNodesResult {
 	return c.metadata.GetShardNodes()
 }
+
+func getProcedureIDPath(rootPath string) string {
```

Comment:
Maybe we should use `/{rootPath}/{clusterName}/{prefixKey}` to make it separate from different clusters.

Reply:
Currently, the procedure design is shared for Ceresmeta and does not involve cluster-level differentiation. However, if you want to add a cluster prefix, the procedure storage would also need to include the prefix.